src/main/java/frc/robot/subsystems/arm/Arm.java

```diff
@@ -2,6 +2,11 @@
 
 import com.revrobotics.CANSparkMax;
 import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
+import frc.robot.Constants;
+import edu.wpi.first.wpilibj2.command.button.CommandGenericHID;
+import edu.wpi.first.wpilibj2.command.button.CommandXboxController;
+
 import java.util.EnumMap;
 
 // extend SubsystemBase class into the Arm class?
```

Comment:
Yes, Arm needs to extend SubsystemBase. You could also consider [PIDSubsystem](https://docs.wpilib.org/en/stable/docs/software/commandbased/pid-subsystems-commands.html#pidsubsystems) if for some reason the motors built-in PID loop is insufficient.